src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -558,19 +558,32 @@ private void AttributeValue(HtmlSymbolType quote)
 
             if (At(HtmlSymbolType.Transition))
             {
-                var valueStart = CurrentLocation;
-                PutCurrentBack();
+                if (NextIs(HtmlSymbolType.Transition))
+                {
+                    // Render a single "@" in place of "@@".
+                    Output(SpanKind.Markup);
+                    AcceptAndMoveNext();
+                    Span.CodeGenerator = SpanCodeGenerator.Null;
+                    Output(SpanKind.Markup, AcceptedCharacters.None);
+                    AcceptAndMoveNext();
+                    Output(SpanKind.Markup, AcceptedCharacters.None);
```

Comment:
- Looks like this is the only significant code change.  I understand #183 was about a parsing error in the context of an attribute value.  Have you confirmed no similar issues exist elsewhere e.g. in strings that are part of directives?  My question isn't a suggestion for additional changes.  Instead I'm hoping we know whether or not there's a cluster of related bugs.
- Unclear why this block calls `Output()` three (four counting the final call in the method) times for two characters.  See the final `Output()` at the bottom of the method is redundant (two successive calls do nothing).  Suspect you need a comment above the first `Output()` mentioning that it outputs the `prefix` (whitespace).